python/example_code/iotthingsgraph/camera.py

```diff
@@ -0,0 +1,177 @@
+#
+# Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+# snippet-sourcedescription:[camera.py demonstrates how to mock an AWS IoT Things Graph camera device in an AWS IoT Greengrass group.]
+# snippet-service:[iotthingsgraph]
+# snippet-keyword:[Python]
+# snippet-keyword:[AWS IoT Things Graph]
+# snippet-keyword:[Code Sample]
+# snippet-sourcetype:[full-example]
+# snippet-sourcedate:[2019-07-25]
+# snippet-sourceauthor:[AWS]
+# snippet-start:[iotthingsgraph.python.camera.complete]
+
+
+import os
+import sys
+import time
+import uuid
+import json
+import logging
+import argparse
+from AWSIoTPythonSDK.core.greengrass.discovery.providers import DiscoveryInfoProvider
+from AWSIoTPythonSDK.core.protocol.connection.cores import ProgressiveBackOffCore
+from AWSIoTPythonSDK.MQTTLib import AWSIoTMQTTClient
+from AWSIoTPythonSDK.exception.AWSIoTExceptions import DiscoveryInvalidRequestException
+
+AllowedActions = ['both', 'publish', 'subscribe']
+
+# General message notification callback
+def customOnMessage(message):
+    print('Received message on topic %s: %s\n' % (message.topic, message.payload))
+
+MAX_DISCOVERY_RETRIES = 10
+GROUP_CA_PATH = "./groupCA/"
+
+# Read in command-line parameters
+parser = argparse.ArgumentParser()
+parser.add_argument("-e", "--endpoint", action="store", required=True, dest="host", help="Your AWS IoT custom endpoint")
+parser.add_argument("-r", "--rootCA", action="store", required=True, dest="rootCAPath", help="Root CA file path")
+parser.add_argument("-c", "--cert", action="store", dest="certificatePath", help="Certificate file path")
+parser.add_argument("-k", "--key", action="store", dest="privateKeyPath", help="Private key file path")
+parser.add_argument("-n", "--thingName", action="store", dest="thingName", default="Bot", help="Targeted thing name")
+
+args = parser.parse_args()
+host = args.host
+rootCAPath = args.rootCAPath
+certificatePath = args.certificatePath
+privateKeyPath = args.privateKeyPath
+clientId = args.thingName
+thingName = args.thingName
+#topic = args.topic
+images = ["https://images-na.ssl-images-amazon.com/images/I/41iz5Tw82IL._AC_US218_.jpg", "https://images-na.ssl-images-amazon.com/images/I/51rMLSWgwRL._AC_US218_.jpg", "https://images-na.ssl-images-amazon.com/images/I/31s6UyPtjOL._AC_US218_.jpg", "https://images-na.ssl-images-amazon.com/images/I/41+K4pC74XL._AC_US218_.jpg"]
```

Comment:
This line is 325 chars long. Can you put each item in the list on a separate line?